nixos/modules/config/users-groups.nix

```diff
@@ -57,9 +57,9 @@ let
     The options {option}`hashedPassword`,
     {option}`password` and {option}`hashedPasswordFile`
     controls what password is set for the user.
-    {option}`hashedPassword` overrides both
-    {option}`password` and {option}`hashedPasswordFile`.
-    {option}`password` overrides {option}`hashedPasswordFile`.
+    {option}`hashedPasswordFile` overrides both
+    {option}`password` and {option}`hashedPassword`.
+    {option}`hashedPassword` overrides {option}`password`.
```

Comment:
Line 57 of the password-option-* test says `hashedPassword -> ... -> password`, so the override is the other way round here, right?

Reply:
Also given that this also depends on `mutableUsers` somehow, I guess this is also worth mentioning.